### ISSUE TITLE
Game highlight on ogre3d.org

### ISSUE THREAD

**Issue by paroj** [RigsOfRods/rigs-of-rods#2340]
If you at some point get the 0.4.8 release out, I could feature it at ogre3d.org like:
https://www.ogre3d.org/2017/12/13/game-highlight-iubes2

would be nice to finally see an Ogre based open source game.

@ulteq @only-a-ptr

**Comment by only-a-ptr** [comment]
> If you release... at some point... 

😆 
Thanks for the offer, much appreciated, RoR could use some highlight.

**Comment by tritonas00** [comment]
The [release](https://github.com/RigsOfRods/rigs-of-rods/releases/tag/2020.01) is out so maybe its time for this @paroj

**Comment by only-a-ptr** [comment]
@tritonas00 @paroj This is almost the time, but let's delay it until the end of February please.
* we have inconsistent GUI and the [remake PR](https://github.com/RigsOfRods/rigs-of-rods/pull/2460) makes things look so much better that it's worth some more wait, The GUI including surveymap is 85% done, I'll fix Dashboads in another PR.
* the [Unicode PR](https://github.com/RigsOfRods/rigs-of-rods/pull/2373) is still not done (just noticed that yesterday) - savegames are wrong.
* most importantly, our main loops are full of duplicate code and suboptimal custom event handling, we positively need to migrate to OGRE's ApplicationContext + SDL before we can let any sensible developer to look at the code.

If I don't make it to 2020-02 release, I give up and you guys decide.

**Comment by tritonas00** [comment]
Outdated, we are two releases ahead already.
